src/LogoFX.Client.Core.Platform/src/CommonProperties.cs

```diff
@@ -1,7 +1,13 @@
-﻿#if NET || NETCORE
+﻿
+#if (NET || NETCORE) && !WINUI3
```

Comment:
@glyad Why do you need the `&& !WINUI3` part ?

Reply:
Fixed